pkg/apis/core/v1alpha1/types.go

```diff
@@ -256,12 +256,14 @@ type CDIConfig struct {
 
 //CDIConfigSpec defines specification for user configuration
 type CDIConfigSpec struct {
-	UploadProxyURLOverride *string `json:"uploadProxyURLOverride,omitempty"`
+	UploadProxyURLOverride   *string `json:"uploadProxyURLOverride,omitempty"`
+	ScratchSpaceStorageClass *string `json:"scratchSpaceStorageClass,omitempty"`
 }
 
 //CDIConfigStatus provides
 type CDIConfigStatus struct {
-	UploadProxyURL *string `json:"uploadProxyURL,omitempty"`
+	UploadProxyURL           *string `json:"uploadProxyURL,omitempty"`
+	ScratchSpaceStorageClass *string `json:"scratchSpaceStorageClass,omitempty"`
```

Comment:
I think shis should be `string` instead of `*string`.  There should always be a value.  '""' is a fine value.

Reply:
Done